static/js/components/UserManagement.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import Chip from "@material-ui/core/Chip";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Typography from "@material-ui/core/Typography";
+import TextareaAutosize from "@material-ui/core/TextareaAutosize";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import PapaParse from "papaparse";
+
+import { showNotification } from "baselayer/components/Notifications";
+
+import * as groupsActions from "../ducks/groups";
+import * as usersActions from "../ducks/users";
+import * as streamsActions from "../ducks/streams";
+import * as inviteUsersActions from "../ducks/inviteUsers";
+
+const sampleCSVText = `example1@gmail.com,1,3,false
+example2@gmail.com,1 2 3,2 5 9,false false true`;
+
+const UserManagement = () => {
+  const dispatch = useDispatch();
+  const { invitationsEnabled } = useSelector((state) => state.sysInfo);
+  const currentUser = useSelector((state) => state.profile);
+  const { allUsers } = useSelector((state) => state.users);
+  const [csvData, setCsvData] = useState("");
+
+  useEffect(() => {
+    const fetchUsers = () => {
+      dispatch(usersActions.fetchUsers());
+    };
+    if (!allUsers?.length) {
+      fetchUsers();
+    }
+  }, [allUsers, dispatch]);
+
+  if (!allUsers?.length || !currentUser?.username?.length) {
+    return (
+      <div>
+        <CircularProgress />
+      </div>
+    );
+  }
+
+  if (!currentUser.acls?.includes("System admin")) {
```

Comment:
Same here, or should also adjust the expected ACLs on the backend

Reply:
Will update

Reply:
Done